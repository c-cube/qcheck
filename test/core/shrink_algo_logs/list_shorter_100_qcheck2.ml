open QCheck2

let list_shorter_100 =
  Test.make ~name:"lists shorter than 100"
    Gen.(list small_int) (Log.shrinks_list_length @@ fun xs -> List.length xs < 100)

let rand = Random.State.make [|1234|];;
try Test.check_exn ~rand list_shorter_100 with Test.Test_fail _ -> ()
