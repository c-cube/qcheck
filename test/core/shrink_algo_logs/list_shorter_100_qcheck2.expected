fails [...] list length: 250
holds [...] list length: 0
fails [...] list length: 125
holds [...] list length: 0
holds [...] list length: 62
holds [...] list length: 93
fails [...] list length: 109
holds [...] list length: 0
holds [...] list length: 54
holds [...] list length: 81
holds [...] list length: 95
fails [...] list length: 102
holds [...] list length: 0
holds [...] list length: 51
holds [...] list length: 77
holds [...] list length: 90
holds [...] list length: 96
holds [...] list length: 99
fails [...] list length: 101
holds [...] list length: 0
holds [...] list length: 50
holds [...] list length: 75
holds [...] list length: 88
holds [...] list length: 94
holds [...] list length: 97
holds [...] list length: 99
fails [...] list length: 100
holds [...] list length: 0
holds [...] list length: 50
holds [...] list length: 75
holds [...] list length: 88
holds [...] list length: 94
holds [...] list length: 97
holds [...] list length: 99
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
fails [...] list length: 100
